app/Http/Controllers/admin/TopicController.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace App\Http\Controllers\admin;
+
+use App\Http\Requests\TopicFormRequest;
+use App\Models\Topic;
+use App\Repositories\Contracts\TopicRepository;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Input;
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Storage;
+
+class TopicController extends Controller
+{
+
+    protected $topic;
+
+    public function __construct(TopicRepository $topic)
+    {
+        $this->topic = $topic;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/admin/TopicController.php</br>**Line**: 35</br>Variable "test_model" is not in valid camel caps format